CMakeLists.txt

```diff
@@ -179,9 +179,7 @@ add_custom_target(pull_matching_test_images
 
 # pull cropped images from saliency images testing folder
 add_custom_target(pull_saliency_test_images
-  COMMAND gdown 1HJLdrm0X3VxnlQ3Z58EE9-Y5VX1NlhAz -O ${CMAKE_BINARY_DIR}/../tests/integration/images/saliency.zip && 
-    mkdir -p ${CMAKE_BINARY_DIR}/../tests/integration/images/saliency &&
-    unzip ${CMAKE_BINARY_DIR}/../tests/integration/images/saliency.zip -d ${CMAKE_BINARY_DIR}/../tests/integration/images/saliency
+  COMMAND gdown 1JvtQUroZJHo51E37_IA2D1mfdJj2smyR --folder -O ${CMAKE_BINARY_DIR}/../tests/integration/images/saliency
```

Comment:
Can you add the line `USES_TERMINAL` to each of these pull_* targets. With this you can see the standard out progress of gdown in real time.

Reply:
Added in https://github.com/tritonuas/obcpp/pull/204/commits/153bdc6ef0a44cd53e7c3697c1d70f708c859706